src/components/InputForm.js

```diff
@@ -0,0 +1,14 @@
+function InputForm() {
+  return (
+    <>
+      <form>
+        <input placeholder="Book title" />
+        <select>
+          <option value="" disabled selected hidden>Author</option>
+        </select>
+      </form>
```

Comment:
- [ ] As required for the project, kindly ensure that your form to create new books has `title` and `author` inputs and a `button` to submit a `Book`. I suggest you change your `<select><option value="" disabled selected hidden>Author</option></select>` to be an `<input />` tag and add the missing button as well.